web/src/components/series.astro

```diff
@@ -0,0 +1,462 @@
+---
+const series_100 = {
+  id: "100",
+  title: "FLARE 100: Core",
+  subseries_list: [
+    {
+      id: "101",
+      title: "101: Python APIs",
+      description: "Learn to use the Client API, Model Controller API, and Job API.",
+      cards: [
+        {
+          title: "Getting Started",
+          tags: ["beg.", "pytorch", "lightning", "sklearn", "tensorflow"],
+          description: "Getting started examples using the Client API, Model Controller API, and Job API for different frameworks.",
+          link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/getting_started"
+        },
+        {
+          title: "Client API",
+          tags: ["docs"],
+          description: "The FLARE Client API provides an easy way for users to convert their centralized, local training code into federated learning code.",
+          link: "https://nvflare.readthedocs.io/en/main/programming_guide/execution_api_type/client_api.html#client-api"
+        },
+        {
+          title: "Model Controller API",
+          tags: ["docs"],
+          description: "The FLARE ModelController API provides an easy way for users to write and customize FLModel-based controller workflows.",
+          link: "https://nvflare.readthedocs.io/en/main/programming_guide/controllers/model_controller.html#model-controller"
+        },
+        {
+          title: "Fed Job API",
+          tags: ["docs"],
+          description: "The FLARE FedJob API allows users to Pythonically define and create job configurations.",
+          link: "https://nvflare.readthedocs.io/en/main/programming_guide/fed_job_api.html",
+          colab_link: "https://colab.research.google.com/github/NVIDIA/NVFlare/blob/main/examples/tutorials/flare_api.ipynb"
+        },
+        {
+          title: "ML/DL to FL",
+          tags: ["beg.", "numpy", "pytorch", "lightning", "tensorflow"],
+          description: "Example for converting Deep Learning (DL) code to Federated Learning (FL) using the Client API. Configurations for numpy, pytorch, lighting, and tensorflow.",
+          link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-world/ml-to-fl"
+        },
+        {
+          title: "Hello FedAvg",
+          tags: ["beg.", "pytorch"],
+          description: "Demonstrate flexibility of the ModelController API, and show how to write a Federated Averaging workflow with early stopping, model selection, and saving and loading.",
+          link: "https://github.com/NVIDIA/NVFlare/tree/main/examples/hello-world/hello-fedavg/README.md"
+        },
+        {
+          title: "Job API Examples",
+          tags: ["int.", "pytorch", "lightning", "sklearn", "tensorflow"],
```

Comment:
int. means "intermediate"? or "integration"?

Reply:
intermediate. These tags are from the tutorial catalog page.